ddtrace/llmobs/_llmobs.py

```diff
@@ -562,8 +593,10 @@ def submit_evaluation(
         :param value: The value of the evaluation metric.
                       Must be a string (categorical), integer (numerical/score), or float (numerical/score).
         """
-        if cls.enabled is False or cls._instance is None or cls._instance._llmobs_eval_metric_writer is None:
-            log.warning("LLMObs.submit_evaluation() requires LLMObs to be enabled.")
+        if cls.enabled is False or cls._instance._llmobs_eval_metric_writer is None:
```

Comment:
```suggestion
        if cls.enabled is False:
```
We don't need to check the writer being None anymore, since we'll always have a default instance.